source/extensions/api_listeners/default_api_listener/api_listener_impl.h

```diff
@@ -133,8 +133,8 @@ class ApiListenerImplBase : public Server::ApiListener,
       }
       void close(Network::ConnectionCloseType) override {}
       void close(Network::ConnectionCloseType, absl::string_view) override {}
-      Network::DetectedCloseType detectedCloseType() const override {
-        return Network::DetectedCloseType::Normal;
+      StreamInfo::DetectedCloseType detectedCloseType() const override {
```

Comment:
You can put a using alias to make your change smaller - but it's already done, so doesn't matter.